src/goto-instrument/nondet_volatile.cpp

```diff
@@ -13,9 +13,13 @@ Date: September 2011
 
 #include "nondet_volatile.h"
 
+#include <util/cmdline.h>
+#include <util/options.h>
 #include <util/std_expr.h>
 #include <util/symbol_table.h>
 
+using havoc_predt = std::function<bool(const exprt &)>;
```

Comment:
:pick: avoid contractions:
```suggestion
using havoc_predicatet = std::function<bool(const exprt &)>;
```

Reply:
Perhaps should_havoc_predicatet? To disambiguate from is_havocked or some other havocking-related question

Reply:
Done